user-guide/modules/ROOT/pages/getting-started-with-linux.adoc

```diff
@@ -79,7 +79,14 @@ cd boost_1_82_0
 +
 The --prefix option specifies the installation directory. You can change this to your preferred location.
 
-. Build and install Boost: `sudo ./b2 install`. This command builds and installs the Boost libraries to the specified prefix directory.
+. Build and install Boost:
```

Comment:
I would only stage `boost-history.adoc` in this PR and include changes to getting started in another PR. The general reason is that they are separate issues. But the circumstantial reason is that's going to conflict with the tabs PR anyway.